<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialsEditorDialog</class>
 <widget class="QDialog" name="MaterialsEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Materials</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="listWidgetMaterials"/>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Material properties</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButtonNew">
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonEditCopy">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Edit a copy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonDelete">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButtonNew</tabstop>
  <tabstop>pushButtonEdit</tabstop>
  <tabstop>pushButtonEditCopy</tabstop>
  <tabstop>pushButtonDelete</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>listWidgetMaterials</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MaterialsEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MaterialsEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
